src/main/java/org/broadinstitute/hellbender/exceptions/UserException.java

```diff
@@ -103,6 +103,12 @@ public CouldNotReadInputFile(String message) {
         public MissingReference(String message) { super(message); }
     }
 
+    public static class MissingIndex extends UserException {
+        private static final long serialVersionUID = 0L;
+
+        public MissingIndex(String message) { super(message); }
```

Comment:
When adding a new exception subclass of `UserException`, it's good practice to insert a standard error message in the constructor. Eg.,

```
public MissingIndex(String file, String message) {
    super(String.format("An index is required but was not found for file %s. %s", file, message)); 
}
```

Reply:
done